src/state/utils.js

```diff
@@ -73,11 +77,23 @@ export const createAsyncThunk = (
         };
       });
 
-      if (dispatchErrorMessage) {
+      if (dispatchErrorMessage && !isAborted) {
         parsedErrors.forEach(message => dispatch(addMessage(message)));
       }
 
       return rejectWithValue({ error, parsedErrors });
     }
   });
 };
+
+export const useFetchData = dataFetchCallback => {
```

Comment:
question: every time this is called, it wraps useCallback. i.e.
```
 useFetchData(useCallback(() => foo
```

Is there a way to write it so you can avoid the extra layer, i.e.
```
 useFetchData(() => foo
```
?

Reply:
@paulschreiber Yes, that is required, react hooks have to be used under the React components directly, this si a good practice and important to validate the ordering of their execution and theri dependencies, so still this is needed not sure if this will be improved in the future.